README.md

```diff
@@ -1,6 +1,6 @@
 # ObjectiveGit
 
-[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage) ![TravisCI](https://travis-ci.org/libgit2/objective-git.svg?branch=master)
```

Comment:
The link is missing. If you click on it, you should be redirected to Travis. Should be:

`[![Build Status](https://travis-ci.org/libgit2/objective-git.svg?branch=master)](https://travis-ci.org/libgit2/objective-git)`